src/wp-includes/option.php

```diff
@@ -361,18 +361,19 @@ function wp_load_alloptions( $force_cache = false ) {
 }
 
 /**
- * Loads and caches certain often requested site options if is_multisite() and a persistent cache is not being used.
+ * Loads and primes caches of certain often requested network options if is_multisite().
  *
  * @since 3.0.0
+ * @since 6.3.0 Also prime caches for network options when persistent object cache is enabled.
  *
  * @global wpdb $wpdb WordPress database abstraction object.
  *
- * @param int $network_id Optional site ID for which to query the options. Defaults to the current site.
+ * @param int $network_id Optional network ID for which network to prime caches of network options. Defaults to the current network.
```

Comment:
```suggestion
 * @param int $network_id Optional. Network ID of network for which to prime network options cache. Defaults to current network.
```

👋🏻 I've read this a few times and keep trying to rewrite it in my head, so here I am. 😄 

* We often start with `Optional.` in docs when a parameter is optional, and then a new sentence. I think it makes sense to change that here.
* It still seems like the word "network" is used too many times, but it is clear. I'm on the fence even with what I've suggested above, but I also like "Network ID for which to prime network options cache." or "ID of network for which to prime..."

Reply:
Good feedback @jeremyfelt ! Committed your suggestion. If I can get approval on the PR, I will commit.